tests/codes/openmc/test_source_params.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: 2021-present M. Coleman, J. Cook, F. Franza
+# SPDX-FileCopyrightText: 2021-present I.A. Maione, S. McIntosh
+# SPDX-FileCopyrightText: 2021-present J. Morris, D. Short
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+import json
+from pathlib import Path
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+from matplotlib.cm import coolwarm
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from bluemira.base.file import get_bluemira_path
+from bluemira.base.parameter_frame import make_parameter_frame
+from bluemira.codes.openmc.params import (
+    OpenMCNeutronicsSolverParams,
+    PlasmaSourceParameters,
+)
+from bluemira.codes.openmc.sources import make_tokamak_source
+
+PARAMS_PATH = Path(get_bluemira_path("codes/openmc", subfolder="tests"), "params.json")
+
+with open(PARAMS_PATH) as j:
+    params = json.load(j)
+openmc_solver_pm_frame = make_parameter_frame(
+    params, OpenMCNeutronicsSolverParams, allow_unknown=True
+)
+pms_params = PlasmaSourceParameters.from_parameterframe(openmc_solver_pm_frame)
+
+
+def test_pedestal_location_in_normalized_radius():
+    assert pms_params.rho_pedestal == 0.94  # [dimensionless]
```

Comment:
Need this on the tests in this module.
```suggestion
    assert pms_params.rho_pedestal == pytest.approx(0.94)  # [dimensionless]
```